server/container/guice/common/src/test/java/org/apache/james/JamesServerExtension.java

```diff
@@ -56,11 +56,22 @@ interface AwaitCondition {
 
     public enum Lifecycle {
         // Restarts the server for each class, including nested classes
-        PER_CLASS(JamesServerExtension::start, (extension, context) -> { }, (extension, context) -> { }, JamesServerExtension::stop),
+        PER_CLASS(
+            (extension, context) -> {
+                extension.setup(context);
+                extension.start(context);
```

Comment:
Can we open this change for the master branch also to see the tests result?

For postgresql branch, only Postgres tests are being executed.

Reply:
@hungphan227 can you take care of this please?

Reply:
https://github.com/apache/james-project/pull/2086